src/utils/acquireSessionId.ts

```diff
@@ -0,0 +1,25 @@
+import { ulid } from 'ulid';
+
+export const SESSION_ID_KEY = 'yext_analytics_sessionId';
+
+/**
+ * Retrieves session id from session storage, or generates a new ULID to use as session id.
+ * The new id is then stored in session storage. Returns null if the sessionStorage API is
+ * unavailable (e.g. The function is running on the server for SSR).
+ */
+export function acquireSessionId(): string | null {
```

Comment:
I sort of prefer `getOrSetupSessionId` since it's crystal clear what it's doing, whereas `acquireSessionId` feels a bit more opaque. What do others think? @wesleyhillyext @nbramblett @AjayBenno @benmcginnis

Reply:
That's fair. I copied this name from `search-ui-react` test-site. but our old search SDK have `getOrSetupSessionId`. I can update the name

Reply:
Yeah I'm curious what others think about the name as well. I'm open to other options!